app/database.py

```diff
@@ -0,0 +1,17 @@
+from dotenv import load_dotenv
+from sqlmodel import create_engine, Session
```

Comment:
Please use PEP-8 style in your imports: https://peps.python.org/pep-0008/#imports